docs/configure.md

```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
 
 ## Configure a storage class
 
-Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster. The configuration details differ depending on your hosting provider, so you should:
+Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster.
 
-1. Create a stub `base/sourcegraph.StorageClass.yaml`.
+Create `base/sourcegraph.StorageClass.yaml` with the appropriate configuration for your cloud provider and commit the file to your fork.
 
-   ```yaml
-   # base/sourcegraph.StorageClass.yaml
-   kind: StorageClass
-   apiVersion: storage.k8s.io/v1
-   metadata:
-     name: sourcegraph
-     labels:
-       deploy: sourcegraph
-   #
-   # The values of the "provisioner" and "parameters" fields will differ depending on the cloud provider that you are using. Please read through https://kubernetes.io/docs/concepts/storage/storage-classes/ in order to know what values to add. 🚨 We recommend specifying SSDs as the disk type if possible. 🚨
-   #
-   # For example, if you are using GKE with a cluster whose nodes are all in the "us-central1-a" zone, you could use the following values:
-   #
-   # provisioner: kubernetes.io/gce-pd
-   # parameters:
-   #  type: pd-ssd
-   #  zones: us-central1-a
```

Comment:
Do you know if this field is now unnecessary?

Reply:
Not necessary, and I tested this as part of 3.1 release testing

![image](https://user-images.githubusercontent.com/754768/53032123-0f7f2000-3423-11e9-9b74-ddcd4818fbb3.png)